### PR TITLE
Task/update student ui for conf subject

### PR DESCRIPTION
**Pull request by wtritch**
Disable the IRIS expando on student name as well as the item viewer, and rubric/exemplar tab on results-by-student if this isn't an IRIS-aware Math or ELA subject
image

student history with a mock subject
image

pull column description text from the subject XML
image

Update student responses with item viewer changes as above
image

### CODE REVIEW

**Comment by jtreuting** [comment]
@wtritch comment on the image before looking at the code.  I thought the icons on the student test history would not show if they didn't exist and would push the text to the left

**Comment by wtritch** [comment]
@jtreuting You are correct, sir.  However, since we just hijacked existing assessments and didn't change their names we're still resolving icons for them.

File: webapp/src/main/webapp/src/app/assessments/assessment-icon.component.ts [hunk 1]
```diff
@@ -172,6 +173,9 @@ export class AssessmentIconComponent {
   @Input()
   styles: any;
 
+  @Output()
+  missingIcon: EventEmitter<boolean> = new EventEmitter(true);
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/assessments/assessment-icon.component.ts]: If the assessment-icon component can't resolve an icon, it announces it.

File: webapp/src/main/webapp/src/app/dashboard/group-dashboard/group-assessment-card.component.html [hunk 2]
```diff
@@ -8,12 +8,14 @@
...
+    <div class="col-xs-4"
+         [hidden]="!hasIcon">
       <div class="icon-wrapper">
-        <assessment-icon [assessment]="measuredAssessment.assessment"></assessment-icon>
+        <assessment-icon [assessment]="measuredAssessment.assessment"
+                         (missingIcon)="hasIcon = !$event"></assessment-icon>
       </div>
     </div>
-    <div class="col-xs-8">
+    <div [ngClass]="hasIcon ? 'col-xs-8' : 'col-xs-12'">
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/dashboard/group-dashboard/group-assessment-card.component.html]: If the icon is missing expand the text to the full column width

File: webapp/src/main/webapp/src/app/dashboard/student-dashboard/student-assessment-card.component.html [hunk 3]
```diff
@@ -8,12 +8,14 @@
...
+    <div class="col-xs-4"
+         [hidden]="!hasIcon">
       <div class="icon-wrapper">
-        <assessment-icon [assessment]="latestExam.assessment"></assessment-icon>
+        <assessment-icon [assessment]="latestExam.assessment"
+                         (missingIcon)="hasIcon = !$event"></assessment-icon>
       </div>
     </div>
-    <div class="col-xs-8">
+    <div [ngClass]="hasIcon ? 'col-xs-8' : 'col-xs-12'">
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/dashboard/student-dashboard/student-assessment-card.component.html]: same expando-on-missing-icon behavior here

**Pull request merged**